### DISCUSSION TITLE
Version 2.6.0

### DISCUSSION THREAD

**Discussion by wmudge** [cloudera-labs/cldr-runner#91] [Announcements]
## What's Changed
* Update to fix pre-commit checks by @wmudge in https://github.com/cloudera-labs/cldr-runner/pull/89
* Update cloudera.exe to 2.4.1 by @wmudge in https://github.com/cloudera-labs/cldr-runner/pull/90

**Full Changelog**: https://github.com/cloudera-labs/cldr-runner/compare/v2.5.1...v2.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/cloudera-labs/cldr-runner/releases/tag/v2.6.0'>Version 2.6.0</a>.</em>

**Comment by wfh-andrew** [comment]
When I pull the latest docker image I get version 2.3.1, were images not created from the newer releases?

```
docker pull ghcr.io/cloudera-labs/cldr-runner:full-latest
```

```
docker image ls
ghcr.io/cloudera-labs/cldr-runner        full-2.3.1   2eaec95b9178  7 months ago   3.2 GB
ghcr.io/cloudera-labs/cldr-runner        full-latest  2eaec95b9178  7 months ago   3.2 GB
```

**Reply by wmudge** [reply]
Hi @wfh-andrew,

We changed the tags in [2.4.0](https://github.com/cloudera-labs/cldr-runner/discussions/83) to accommodate the pull policy with `ansible-navigator` (see #80).

If you want to pull the latest `full` image, use `docker pull ghcr.io/cloudera-labs/cldr-runner-full:latest`. See [cldr-runner-full packages](https://github.com/cloudera-labs/cldr-runner/pkgs/container/cldr-runner-full) for all the published images for this new labelling scheme.

Thanks!

**Reply by wfh-andrew** [reply]
Thank you @wmudge, that works!
